src/test/java/com/defano/hypertalk/ast/expression/function/VersionFuncTest.java

```diff
@@ -0,0 +1,40 @@
+package com.defano.hypertalk.ast.expression.function;
+
+import com.defano.hypertalk.GuiceTest;
+import com.defano.hypertalk.ast.expression.Expression;
+import com.defano.hypertalk.ast.model.Value;
+import com.defano.hypertalk.exception.HtSemanticException;
+import com.google.common.collect.Lists;
+import org.junit.Ignore;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class VersionFuncTest extends GuiceTest<VersionFunc> {
+
+    private Expression mockExpression = Mockito.mock(Expression.class);
```

Comment:
Is this necessary? I would not expect you to need a mocked expression in this test?

Reply:
No, we don't need it here. Copy and paste mistake.
![image](https://user-images.githubusercontent.com/3395566/169669230-5d82a990-5a40-4ccb-ac0d-e3d838e75b8b.png)